pypxlib/__init__.py

```diff
@@ -24,12 +25,27 @@ def next(self):
 		# Python 3:
 		__next__ = next
 
-	def __init__(self, file_path, encoding='cp850'):
+	def __init__(self, file_path, blob_file_path=None, encoding='cp850'):
 		self.file_path = file_path
+		self.blob_file_path = blob_file_path
 		self.encoding = encoding
 		self.pxdoc = PX_new()
 		if PX_open_file(self.pxdoc, file_path.encode(self.PX_ENCODING)) != 0:
 			raise PXError('Could not open file %s.' % self.file_path)
+
+		# BLOB Support
+		# http://pxlib.sourceforge.net/documentation.php?manpage=PX_set_blob_file
+		if not blob_file_path:
+			# If not blob_file_path given, we try to acces automatically the related MB file
+			possible_blob_file = str(file_path).replace('.db', '.mb').replace('.DB', '.MB')
```

Comment:
That was just for ensure that replace was callable from `file_path`. It's unnecessary 👍